tasks/breeze/generators/templates/breeze_config.ecr

```diff
@@ -6,7 +6,7 @@ Breeze.configure do |settings|
   settings.database = AppDatabase
 
   # Enable Breeze only for this environment
-  settings.enabled = LuckyEnv.development?
+  settings.enabled = !LuckyEnv.task? && LuckyEnv.development?
```

Comment:
When breeze is required before running migrations, the QueryEvent from pulsar is logged during migrations which throws an error because breeze tables don't exist yet. Migrations are ran in a task, so this fixes it.